pos/.gitignore

```diff
@@ -0,0 +1,22 @@
+# dependencies
+/node_modules
+
+# next.js
+/.next/
+
+# local env files
+.env*.local
+.env
+
+# typescript
+*.tsbuildinfo
+next-env.d.ts
+
+.vscode
```

Comment:
This settings should very probably be in your `~/.gitignore` as it's proper to your work environment and not to the project or its language.